Makefile

```diff
@@ -1127,9 +1128,10 @@ clean:
 	rm -rf $(TMP_DIR)
 	rm -rf $(FILTERS_DIR)
 	rm -rf $(INCLUDE_DIR)
-	rm -rf $(DOC_DIR)
+	rm -rf $(SHARE_DIR)
 	rm -rf $(DISTRIB_DIR)
 	rm -rf $(ROOT_DIR)/apps/*/bin
+	rm -f Doxyfile
```

Comment:
If Doxyfile is a build intermediate (which I guess it is now?) then it should go in TMP_DIR

Reply:
Done